src/main/java/userstuff/Ui.java

```diff
@@ -0,0 +1,64 @@
+package userstuff;
+
+import java.util.Scanner;
+
+/**
+ * A class to handle the Ui of chatbot.
+ */
+public class Ui {
+
+    /** The Scanner object used by this ui to read input. */
+    Scanner scanner;
+
+    /**
+     * Initialises the scanner field variable.
+     */
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Draws a line separating each conversation.
+     *
+     */
+    public void showLine() {
+        System.out.println("\t____________________________________________________________");
+    }
+
+    /**
+     * Greets the user of chatbot.
+     */
+    public void showWelcome() {
+        this.showLine();
+        System.out.println("\tHello I am Vishnu.");
+        System.out.println("\tWhat can I do for you?");
+        this.showLine();
+    }
+
+    /**
+     * Says bye to the user.
+     */
+    public void showBye() {
+        System.out.println("\tBye. Hope to see you again soon!");
+        this.showLine();
```

Comment:
since `showLine` is not being overshadowed, perhaps we can omit replace `this.showLine()` with just `showLine()`?